sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -36,3 +39,8 @@ def iterable():
 @streams_api.route('/error', methods=['GET'])
 def error():
     return Response(stream_json_error(), status=400)
+
+@streams_api.route('/compressed', methods=['GET'])
+def compressed():
+    return Response(stream_compressed_header_error(), status=300, headers={"Content-Type": "gzip"})
```

Comment:
I believe the header passed should be "Content-Encoding" here instead of "Content-Type". @iscai-msft - can you give me a sanity check? :)

Reply:
Done!